react/features/base/connection/reducer.ts

```diff
@@ -57,6 +58,11 @@ ReducerRegistry.register<IConnectionState>(
         case SET_LOCATION_URL:
             return _setLocationURL(state, action);
 
+        case SET_PREFER_VISITOR:
+            return assign(state, {
+                preferVisitor: action.preferVisitor
```

Comment:
I see we are setting `preferVisitor` but I'm having a hard time to see how do we use it. Do you use it somewhere?